### DISCUSSION TITLE
Is wxt production ready?

### DISCUSSION THREAD

**Discussion by EmilMalanczak** [wxt-dev/wxt#439] [Q&A] [answered]
I'm currently developing an extension with Plasmo and thinking about migration to wxt. 
The content-ui part is the most interesting part - what are the current limitations?

**Comment by aklinker1** [comment, marked as answer]
Yes, WXT is production ready. There are some medium/large sized extensions using it right now, scroll down on the homepage (https://wxt.dev). There are a few APIs that haven't been finalized, so I haven't released `v1.0`. I don't expect any large breaking changes left before then `v1.0`.

The main difference between WXT's content script UI and plasmo's is that plasmo manages the UI's life cycle, and will automatically mount/unmount the UI when the DOM changes. In WXT, you have to manage that yourself.

**Reply by EmilMalanczak** [reply]
Thanks for the response! That's a good thing for me so even better, I'll migrate to `wxt` soonish
